metabench.cmake

```diff
@@ -147,10 +147,13 @@ endfunction()
 #       The path to a JSON file containing data for rendering the chart in the
 #       format specified by NVD3.
 #
+#   [NAME name]:
+#       A name used for the output files. Defaults to `target`.
```

Comment:
This could be made more precise by saying something like "A name used for the output files. The JSON and HTML files generated will be named ${name}.json and ${name}.html, respectively. Defaults to `target`."